classes/2.html

```diff
@@ -113,13 +113,120 @@ <h6>Atributos de un elemento HTML</h6>
             </li>
           </ul>
           <pre><code data-line-numbers data-trim class="lang-html">
-        <p class="access-text"> Esto contiene un párrafo</p>
+        <p class="access-text">Esto contiene un párrafo</p>
       </code></pre>
         </section>
 
         <section>
           <h6>Anidación de elementos</h6>
+          <ul>
+            <li>
+              Los elementos se pueden anidar, es decir, colocar elementos dentro de otro elemento.
+            </li>
+            <li>
+              Se recomienda siempre cerrar los tags aninados en orden inverso al que comienzan.
+              Es decir, en el ejemplo debajo, el tag &lt;em&gt cierra primero ya que fue el último tag en comenzar. 
+            </li>
+          </ul>
+          <pre><code data-line-numbers data-trim class="lang-html">
+            <strong>My bold text and <em>my bold and emphasized text</em></strong>
```

Comment:
```suggestion
            <strong>Mi texto en negritas <em>mi texto en negritas y enfatizado</em></strong>
```

Queda algo raro igual 😅